sample_files/.travis.yml

```diff
@@ -14,8 +14,10 @@ env:
   - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
   - TRANSIFEX_USER='transbot@odoo-community.org'
   # This line contains the encrypted transifex password
-  # To encrypt transifex password, use
-  # $ travis encrypt TRANSIFEX_PASSWORD=your-password -r owner/project
+  # To encrypt transifex password, install travis ruby utils with:
+  #   $ [sudo] gem install travis
```

Comment:
The standard for displaying a sudo command is to use `#` as the prompt instead of `$`
Is it a good idea to suggest sudo?

Some might have travis (Trajectory Analyzer and Visualizer) accidentally installed. I know because I did.
Might be better to suggest installing it as the local user and calling it from where gem installs it.